force-app/main/triggers/classes/HOT_RequestHandler.cls

```diff
@@ -389,4 +399,12 @@ public without sharing class HOT_RequestHandler extends MyTriggers {
             }
         }
     }
+
+    public static void setArchiveAsOwner(List<Hot_Request__c> requests) {
+        QueueSobject archive = [SELECT QueueId FROM QueueSobject WHERE Queue.Name = 'HOT Arkiv' LIMIT 1];
+
+        for (HOT_Request__c request : requests) {
+            request.OwnerId = archive.QueueId;
+        }
+    }
```

Comment:
Skulle vi laget en felles metode i HOT_Utility.cls, som fungerer for alle SOBJECTS?